database/java/2012/2098.yaml

```diff
@@ -0,0 +1,53 @@
+---
+cve: CVE-2012-2098
+title: CVE in org.apache.commons:commons-compress
+description: >
+    Algorithmic complexity vulnerability in the sorting algorithms in bzip2 compressing stream (BZip2CompressorOutputStream) in Apache Commons Compress before 1.4.1 allows remote attackers to cause a denial of service (CPU consumption) via a file with many repeating inputs.
+cvss_v2: 5.0
+references:
+    - http://ant.apache.org/security.html
+    - http://archives.neohapsis.com/archives/bugtraq/2012-05/0130.html
+    - http://commons.apache.org/compress/security.html
+    - http://lists.fedoraproject.org/pipermail/package-announce/2012-June/081697.html
+    - http://lists.fedoraproject.org/pipermail/package-announce/2012-June/081746.html
+    - http://lists.fedoraproject.org/pipermail/package-announce/2013-May/105049.html
+    - http://lists.fedoraproject.org/pipermail/package-announce/2013-May/105060.html
+    - http://osvdb.org/82161
+    - http://packetstormsecurity.org/files/113014/Apache-Commons-Compress-Apache-Ant-Denial-Of-Service.html
+    - http://secunia.com/advisories/49255
+    - http://secunia.com/advisories/49286
+    - http://www-01.ibm.com/support/docview.wss?uid=swg21644047
+    - http://www.securityfocus.com/bid/53676
+    - http://www.securitytracker.com/id?1027096
+    - https://exchange.xforce.ibmcloud.com/vulnerabilities/75857
+
+affected:
+    - groupId: org.apache.commons
+      artifactId: commons-compress
+      version:
+        - ">= 1.5 & <= 1.8.3"
```

Comment:
Victims uses kinda weird syntax for version ranges. If you need to express that all versions between 1.5 and 1.8.3 are affected, then you would say something like: `"<=1.8.3,1.5"`

Reply:
However, looking at the description of this vulnerability, it says `in Apache Commons Compress before 1.4.1 allows remote attackers`. Thus the affected versions should be `"<=1.4"` and fixedin `">=1.4.1"` (?) Or I am missing something here :)